{"uid":"6a3fe77c38283c7f","name":"SyncManager sync throws error when documents sync fails","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager:.frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager#SyncManager sync throws error when documents sync fails","time":{"start":1747739062000,"stop":1747739062035,"duration":35},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":2},"items":[{"uid":"5c4d748416e73793","reportUrl":"https://input-output-hk.github.io/catalyst-voices/allure-action/fix_recent_proposals/test-report/15135605366_1747740261018/#testresult/5c4d748416e73793","status":"passed","time":{"start":1747739062000,"stop":1747739062035,"duration":35}}]},"tags":[]},"source":"6a3fe77c38283c7f.json","parameterValues":[]}